### PR TITLE
Fix the spyplane not having a tooltip ingame

### LINKED ISSUE #11128: Spy Plane has no name tag
Hey guys, noticed on christmass release, the Spy PLane itself has no name describing it ingame.

spyplane

### PR DESCRIPTION
**Pull request by abcdefg30**
Fixes #11128.

### CODE REVIEW

File: mods/ra/rules/aircraft.yaml [hunk 1]
```diff
@@ -332,7 +332,7 @@ U2:
 	Inherits: ^Plane
 	Health:
 		HP: 2000
-	EditorOnlyTooltip:
+	Tooltip:
 		Name: U2 Spy Plane
```
Comment (pchote) [inline mods/ra/rules/aircraft.yaml]: The "U2" seems unnecessary / unpolished here (it is nothing like a real U2).  Can we change this to "Spy Plane" while we're at it?

**Comment by abcdefg30** [comment]
Changed the tooltip.

**Comment by pchote** [comment]
:+1:

**Comment by obrakmann** [comment]
:+1:

**Comment by obrakmann** [comment]
[Changelog](https://github.com/OpenRA/OpenRA/wiki/Changelog/_compare/ef081102f57310ea51f8e83b870a5d8327ac4ae4)

**Pull request merged**